### PR TITLE
Add Table Store 0.2.0 release

### PR DESCRIPTION
**Pull request by JingsongLi**

### CODE REVIEW

File: _posts/2022-08-08-release-table-store-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+The Apache Flink community is pleased to announce the release of the
+[Apache Flink Table Store](https://github.com/apache/flink-table-store) (0.2.0).
+
+Please check out the full [documentation]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.2/) for detailed information and user guides.
+
+<br/>
+
+## What is Flink Table Store
+
+Flink Table Store is a data lake storage for streaming updates/deletes changelog ingestion and high-performance queries in real time.
```
Comment (carp84) [inline _posts/2022-08-08-release-table-store-0.2.0.md:29]: It seems the description in this blog post is different from the readme file of the [master branch](https://github.com/apache/flink-table-store/blob/master/README.md) and to-be-released source package, and suggest keeping them in sync.

File: _posts/2022-08-08-release-table-store-0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+ Spark     | 3.2      | read        | Projection, Filter |
+ Spark     | 3.3      | read        | Projection, Filter |
+ Trino     | 358      | read        | Projection, Filter |
+ Trino     | 388      | read        | Projection, Filter |
+
+<br/>
+
+### Append-only
+
+Append-only tables are a performance feature that only accepts INSERT_ONLY data to append to the storage
```
Comment (carp84) [inline _posts/2022-08-08-release-table-store-0.2.0.md]:
```suggestion
The append-only table feature is a performance improvement and only accepts INSERT_ONLY data to append to the storage
```

File: _config.yml [hunk 3]
```diff
@@ -720,9 +736,9 @@ release_archive:
         release_date: 2022-04-02
 
     flink_table_store:
-      - version_short: 0.1
-        version_long: 0.1.0
-        release_date: 2022-05-11
+      - version_short: 0.2
+        version_long: 0.2.0
+        release_date: 2022-08-08
```
Comment (carp84) [inline _config.yml]: Minor: please remember to update the date accordingly

File: _posts/2022-08-08-release-table-store-0.2.0.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "Apache Flink Table Store 0.2.0 Release Announcement"
+subtitle: "For building dynamic tables for both stream and batch processing in Flink, supporting high speed data ingestion and real-time data query."
+date: 2022-08-08T08:00:00.000Z
```
Comment (carp84) [inline _posts/2022-08-08-release-table-store-0.2.0.md]: Minor: please remember to update the date accordingly

**Review by carp84 — APPROVED** [review]
Overall LGTM, just some minor suggestions/comments.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**